xga/utils.py

```diff
@@ -694,7 +702,10 @@ def check_telescope_choices(telescope: Union[str, List[str]]) -> List[str]:
 # This checks for an installation of Ciao
 CALDB_AVAIL = False
 
-if 'not installed' in split_out[5].lower():
+if split_out == '':
```

Comment:
So rather than this if addition and extra warning, please change the 

```python
elif 'not installed' in split_out[5].lower():
```

to 

```python
if CIAO_VERSION is not None and 'not installed' in split_out[5].lower():
```

Reply:
See reply to comment above (line 687)